.goreleaser.yaml

```diff
@@ -10,6 +10,8 @@ builds:
       - CGO_ENABLED=0
     ldflags:
       - -X "main.Version={{.Env.GORELEASER_CURRENT_TAG}}"
+      - -X "main.MixPanelEndpoint=https://api.mixpanel.com"
+      - -X "main.MixPanelToken=4bfb2414ab973c741b6f067bf06d5575"
```

Comment:
I think we already discussed this being public, but just to clarify, is there an alternative way to prevent people from getting this token and populating our metrics with malformed data?

Reply:
One alternative here would be to have an additional API where we simply accept a http POST of something like this:

```
{
 "uid": "abc123",
 "event": "eventName"
)
```

On the other hand, it's quite common to have analytics tokens available on the user side. Their own documentation suggests the token will be in view of the users.